src/main/java/page_objects/MyTicketPage.java

```diff
@@ -1,4 +1,33 @@
 package page_objects;
 
+import helpers.DriverHelper;
+import helpers.ElementHelper;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
 public class MyTicketPage extends BasePage {
+
+    //Locators
+    private By btnCancel = By.cssSelector("input[value='Cancel']");
+    private By rowTicket = By.xpath("//table[@class='MyTable']/tbody//following-sibling::tr");
+
+    //Elements
+    private WebElement getBtnCancel() {
+        return DriverHelper.getDriver().findElement(btnCancel);
+    }
+
+    //Methods
+    public void clickCancelButon() {
+        getBtnCancel().click();
+    }
+
+    public void cancelTicket() {
+        ElementHelper.scrollToView(getBtnCancel());
+        clickCancelButon();
+        DriverHelper.getDriver().switchTo().alert().accept();
+    }
+
+    public boolean isTicketDeleted() {
```

Comment:
do not write the boolean like this, this returns `false` if the ticket is not displayed

Reply:
I call it from method 
 public static boolean isElementPresented(By locator) {
        try {
            DriverHelper.getDriver().findElement(locator);
            return true;
        } catch (NoSuchElementException e) {
            return false;
        }